### ISSUE TITLE
Add information gathering options to registry request form.

### ISSUE THREAD

**Issue by jleach** [bcgov/platform-services-registry#79]
On the request form, where teams enter information about their project, gather some additional information about what services the project uses. This information will be used in reporting to better understand what teams need and what features they use. The first team to use this information will be the shared components team.

**Comment by jleach** [comment]
Waiting on J-P to give is a list of services / things that we should include in the web form.

**Comment by jleach** [comment]
Duplicate of #76
